buildlib/azure-pipelines-release.yml

```diff
@@ -41,3 +41,21 @@ stages:
               assets: |
                 ./ucx-*.tar.gz
                 ./rpm-dist/ucx-*.src.rpm
+
+          - bash: |
```

Comment:
1. Now you are doing `automake` & `configure` twice.
2. Disagree with the `status=Failure`. Why not skipped? BTW, this step is better to move into the template.